docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,239 @@ The ACS will be used for V&V testing. It is defined as:
+  * - Joint Filer 
+    - This row through 'dependent filer' are to be included if there is a joint filer ONLY 
+  * - First name 
+    - 
+  * - Middle initial 
+    - 
+  * - Last name 
+    - 
+  * - Age
+    - 
+  * - DOB
+    -  
+  * - Mailing Address
+    -  
+  * - Social Security Number 
+    -
+  * - Income 
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Employer ID
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Employer Address 
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Employer Zip Code 
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Dependent
+    - This row through the end are to be included for each dependent on the tax filing 
+  * - First name 
+    - 
+  * - Middle initial 
+    - 
+  * - Last name 
+    - 
+  * - Age 
+    -  
+
+
+
+**Who to Sample** 
+
+Not everyone who receives a W2 or 1099 will end up filing taxes. 
+However, those who do not are concentrated in low incomes for whom 
+taxes are not required. Currently, we will chose to have all those 
+who are legally required to file taxes, file taxes. This is a 
+limitation and is listed below. 
+
+For simulants that receive below the minimum income, 42.14% will 
+still file taxes. [Cilke_1998]_ The remainder will not. The minimum 
+income is based on the household structure and if listed in the table below. 
+We will not model persistance year to year. 
+
+**In the current model, no one will be low income, this will be changed later.** 
+
+.. list-table:: Minimum Income  
+  :widths: 20 20 
+  :header-rows: 0
+
+  * - Simulant Type 
+    - Minimum Income 
+  * - Single filing, under 65 
+    - $12,550 
+  * - Single filing, over 65 
+    - $14,250
+  * - Married joint filing, both under 65 
+    - $25,100
+  * - Married joint filing, one under 65 
+    - $26,450
+  * - Married joint filing, both over 65 
+    - $27,800
+  * - Married separate filing 
+    - $5
+
+
+Based on the household structure, the following rules can be applied 
+for who files taxes: 
+
+- Assume that 95% of spouses file jointly, this can be randomly assigned. [Nolo]_ Others will file separately. 
+    * The only spouses we will recognize are [Reference person, Opp-sex spouse] and [Reference person, Same-sex spouse]. 
+    * The reference person will submit the form, the spouse will be listed as the joint filer. 
+    * There does not need to be persistance in who files jointly, it can be re-randomly drawn each year. 
+- All other non-married simulants in a household with a W2 or 1099 will file separately, based on the income rules above (e.g., a low-income earner in a house with other earners will be randomly assigned to file or not file independent of others in the household) 
```

Comment:
```suggestion
- All other non-married simulants in a household with a W2 or 1099 will file separately, based on the income rules above (e.g., a low-income earner in a house with other earners will be randomly assigned to file or not file, independent of others in the household) 
```

I first read this as whether or not they would file independently, i.e. as a dependent or not.

Reply:
Also, it is not clear to me how this interacts with claiming dependents. If someone has a W2 but is below the $4,300 number for being a dependent (and also meets the other criteria for being a dependent), what happens?

Reply:
I added something to clarify - but I think the confusion is that you can BOTH file taxes for yourself AND be claimed as a dependent. Imagine a high schooler who worked a summer job. They likely would get some money back if they file taxes, but also can be claimed as a dependent. I am basically assuming that everyone maximizes their tax benefits and so is claimed as a dependent if they can be, but some also will choose to file their own taxes.